dao/service.go

```diff
@@ -0,0 +1,20 @@
+package dao
+
+import "github.com/companieshouse/emergency-auth-code-api/config"
+
+// Service interface declares how to interact with the persistence layer regardless of underlying technology
+type Service interface {
+	// CompanyHasAuthCode will find a single payable resource with the given companyNumber and reference
```

Comment:
Change this comment to something like `// CompanyHasAuthCode will check if this company has an existing auth code`